apps/fg_http/lib/fg_http_web/views/device_view.ex

```diff
@@ -1,3 +1,13 @@
 defmodule FgHttpWeb.DeviceView do
   use FgHttpWeb, :view
+
+  alias FgHttp.Devices.Device
+
+  def rules_title(%Device{} = device) do
+    num_rules = length(device.rules)
+
+    "rule"
+    |> Inflex.inflect(num_rules)
+    |> (&("#{num_rules} " <> &1)).()
```

Comment:
would extracting this line into a new function be useful for readability?
```suggestion
    |> concatenate_with(num_rules)
```

Reply:
Good suggestion!